### PR TITLE
mtdjr userspace changes and Nyquist keymap addition

### PR DESCRIPTION
**Pull request by mtdjr**

### CODE REVIEW

File: keyboards/nyquist/keymaps/mtdjr/config.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/nyquist/keymaps/mtdjr/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

File: keyboards/nyquist/keymaps/mtdjr/config.h [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#define CONFIG_USER_H
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/nyquist/keymaps/mtdjr/config.h]: This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.

File: keyboards/nyquist/keymaps/mtdjr/rules.mk [hunk 3]
```diff
@@ -0,0 +1,6 @@
+RGBLIGHT_ENABLE = yes
+BACKLIGHT_ENABLE = yes
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/nyquist/keymaps/mtdjr/rules.mk]: Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.

File: keyboards/nyquist/keymaps/mtdjr/config.h (hunk 2 above)
Comment (mtdjr) [reply keyboards/nyquist/keymaps/mtdjr/config.h]: done

File: keyboards/nyquist/keymaps/mtdjr/rules.mk (hunk 3 above)
Comment (mtdjr) [reply keyboards/nyquist/keymaps/mtdjr/rules.mk]: done

File: keyboards/nyquist/keymaps/mtdjr/config.h (hunk 1 above)
Comment (mtdjr) [reply keyboards/nyquist/keymaps/mtdjr/config.h]: done

**Pull request merged**